config/application.rb

```diff
@@ -17,5 +17,12 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+    # Double-output logging, both to Rails.logger and STDOUT.  Helps avoid puts statements.
```

Comment:
i think you could remove "helps avoid puts statements" and just leave that in the PR description